openai/openai_test.go

```diff
@@ -87,6 +88,27 @@ func TestMiddleware(t *testing.T) {
 				assert.Equal(t, "Test Model", listResp.Data[0].Id)
 			},
 		},
+		{
+			Name:       "OpenAI Retrieve Handler",
+			Method:     http.MethodGet,
+			Path:       "/api/show/:model",
+			TestPath:   "/api/show/test-model",
+			Middleware: RetrieveMiddleware,
+			Endpoint: func(c *gin.Context) {
+				c.JSON(http.StatusOK, api.ShowResponse{
+					ModifiedAt: time.Date(2024, 6, 17, 13, 45, 0, 0, time.UTC),
+				})
+			},
+			Expected: func(t *testing.T, resp *httptest.ResponseRecorder) {
+				var retrieveResp Model
+				if err := json.NewDecoder(resp.Body).Decode(&retrieveResp); err != nil {
+					t.Fatal(err)
+				}
+
+				assert.Equal(t, "model", retrieveResp.Object)
```

Comment:
Let's not use `assert` either – just `t.Fatal` `t.Fatalf` etc in the standard `testing` library is best